pkg/controller/bootstrap/bootstrap.go

```diff
@@ -147,7 +149,17 @@ func (b *Bootstrap) Run(destDir string) error {
 	if cconfig == nil {
 		return fmt.Errorf("error: no controllerconfig found in dir: %q", destDir)
 	}
-	iconfigs, err := template.RunBootstrap(b.templatesDir, cconfig, psraw, featureGate)
+
+	if featureGate == nil {
+		return fmt.Errorf("error: no featuregate found in dir: %q", destDir)
+	}
```

Comment:
Installer will now always create a feature gate manifest, the MCO now has to depend on that to make decisions during the bootstrap